docs/docs/60-new-docs/50-user-guide/60-reference-docs/50-analysis-templates.md

```diff
@@ -0,0 +1,216 @@
+---
+sidebar_label: Analysis Templates
+description: Learn about AnalysisTemplate for verification
+---
+
+# Analysis Template Reference
+
+An `AnalysisTemplate` is a resource that defines how to perform verification testing, including:
+* container images and commands to run
+* queries to external monitoring tools
+* how to interpret results from metric providers
+* success or failure criteria
+* frequency and duration of measurements
+
+`AnalysisTemplate` resources (and the `AnalysisRun` resources that are spawned from them) are CRDs re-used from the [Argo Rollouts](https://argoproj.github.io/argo-rollouts) project. They were intentionally built to be useful in contexts other than Argo Rollouts. Re-using this resource type to define verification processes means those processes benefit from this rich and battle-tested feature of Argo Rollouts.
+
```

Comment:
Due to a somewhat common point of confusion among users, I think we need to explicitly say _somewhere_ that Kargo's re-use of Rollouts types or this purpose does not in any way force users to use `Rollout`s in place of `Deployment`s.

I'm wondering if this is a good spot for that or if there's a more prominent spot where that disclaimer might fit.

Reply:
Or maybe it belongs in the FAQ actually.